src/main/cms/cms_menu_ledstrip.c

```diff
@@ -61,7 +61,7 @@ const char * const ledProfileNames[LED_PROFILE_COUNT] = {
     "RACE",
     "BEACON",
 #ifdef USE_LED_STRIP_STATUS_MODE
```

Comment:
Should this not be `USE_LED_STRIP_NORMAL_MODE`?

Reply:
Fixed, thanks.